server_admin/topics.adoc

```diff
@@ -33,7 +33,7 @@ include::topics/clients/oidc/proc-using-a-service-account.adoc[]
 include::topics/clients/oidc/audience.adoc[]
 include::topics/clients/saml/proc-creating-saml-client.adoc[]
 include::topics/clients/saml/idp-initiated-login.adoc[]
-include::topics/clients/saml/entity-descriptors.adoc[]
+include::topics/clients/saml/proc-entity-descriptors.adoc[]
```

Comment:
Following the gerund-noun format for a procedure, this should be proc-using-an-entity-descriptor.adoc